flink-cyber/flink-common/src/main/java/com/cloudera/cyber/flink/Utils.java

```diff
@@ -221,6 +224,19 @@ public static Configuration getConfiguration() {
         return ConfigHolder.INSTANCE;
     }
 
+    public static boolean timeCompare(Long unit1, String unitType1, Long unit2, String unitType2) {
+        if (unit1 == null && unitType1 == null && unit2 == null && unitType2 == null) {
+            return true;
+        }else if (unit1 == null || unitType1 == null || unit2 == null || unitType2 == null) {
+            return false;
+        }
+        return Time.of(unit1, TimeUnit.valueOf(unitType1)).toMilliseconds() == Time.of(unit2, TimeUnit.valueOf(unitType2)).toMilliseconds();
+    }
```

Comment:
it might be better to either rename to timeEquals, or return int value as in Comparable interface. This can be easily achieved by subtracting one value from another.
Also, might be better to shorten the expression by using TimeUnit.toMillis(long)